packages/contracts-ops/scripts/updateConfig.ts

```diff
@@ -0,0 +1,77 @@
+import * as config from "@nomad-xyz/configuration";
```

Comment:
I like this but think it needs more development. what are the common actions, and do we have standard formats for them?

I like having the implementations interface 🤔